app/src/test/java/com/notifier/app/core/data/networking/HttpClientFactoryTest.kt

```
@@ -0,0 +1,71 @@
+
+        assertThat(client).isInstanceOf(HttpClient::class.java)
+    }
+
+    @Test
+    fun testHttpClientFactory_loggingIsInstalled() {
+        val client = createMockClient()
+
+        val isLoggingInstalled = try {
+            client.plugin(Logging)
+            true
+        } catch (e: Throwable) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)